cli/src/cli.rs

```diff
@@ -82,6 +85,30 @@ where
                         println!("[error] {}\n", e);
                     }
                 },
+                Command::ExecuteFromFile(filename) => {
+                    if let Err(e) = self.load(&filename) {
+                        println!("[error] {}\n", e);
+                    }
+                }
+            }
+        }
+
+        Ok(())
+    }
+
+    pub fn load(&mut self, filename: &str) -> Result<()> {
+        let mut sqls = String::new();
+        // We open the file
+        File::open(filename)?.read_to_string(&mut sqls)?;
+        // We get all the sql query
+        for sql in sqls.split(";\n") {
+            if sql.trim().len() > 0 {
+                match self.glue.execute(&(sql.to_owned() + ";")) {
```

Comment:
In the cli app, we use `;` to recognize the command.
However, for `glue.execute`, `;` at the end of line is optional.

```rust
self.glue.execute(sql)
```
would be enough